blueflood-core/src/test/java/com/rackspacecloud/blueflood/utils/GlobPatternTest.java

```diff
@@ -0,0 +1,144 @@
+package com.rackspacecloud.blueflood.utils;
+
+import org.junit.Assert;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.util.regex.PatternSyntaxException;
+
+public class GlobPatternTest {
```

Comment:
:+1 for doing this!  Can we add some more descriptive names than testGlobToRegex1, testGlobToRegex2, etc.  I'd do something along the lines of GlobToRegexPeriodStar, GlobToRegexEmpty, etc.